CMakeLists.txt

```diff
@@ -1245,16 +1245,10 @@ if(VRPN_BUILD_SERVER_LIBRARY)
 	if(VRPN_INSTALL)
 		install(TARGETS
 			vrpnserver
-			ARCHIVE
-			DESTINATION
-			lib
-			COMPONENT
-			serversdk
-			PUBLIC_HEADER
-			DESTINATION
-			include
-			COMPONENT
-			serversdk)
+			RUNTIME DESTINATION bin COMPONENT serversdk
+			LIBRARY DESTINATION lib${LIB_SUFFIX} COMPONENT serversdk
+			ARCHIVE DESTINATION lib${LIB_SUFFIX} COMPONENT serversdk
```

Comment:
Maybe... I don't think I've ever actually seen GNUInstallDirs used, but from some poking around it looks like GNUInstallDirs is newer and "meant" to replace LIB_SUFFIX. (Doing a Google search on both, LIB_SUFFIX is more popular, with 18k hits vs. 3k.)

That said, note that you *can* set LIB_SUFFIX to things like `/x86_64` (i.e. `DESTINATION lib/x86_64`), not just `64`; it's only insufficient if first path component of the correct location doesn't start with (or equal) "lib". I'm not sure that isn't adequate for Debian...

(I'll also note that a major reason for this change was because I was poking it anyway to make shared work, and it seemed silly to not add a `${LIB_SUFFIX}` while I was at it. Using GNUInstallDirs would have been more invasive... which is not to say it wouldn't make sense for a follow-up change.)